doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -606,6 +606,10 @@ Networking samples
 
   * Fixed not to fail with a fatal error if IPv4 or IPv6 server setup fails.
 
+* :ref:`aws_iot` sample:
+
+  * Added support for the :ref:`zephyr:nrf54l15dk_nrf54l15` board in combination with an nRF7002-based shield.
```

Comment:
We could have some information about the nRF54l15 support in the introduction section, since we talk about other devices as well - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/net/aws_iot/README.html.

Also, the configuration files section would require an update about the nRF54l15 conf and overlay files.